### ISSUE TITLE
Warning message spam "please use `truncation=True` to explicitely truncate examples to max length"

### ISSUE THREAD

**Issue by PhilipMay** [deepset-ai/FARM#525]
Hi,

since I use the code from master branch I am getting spammed with these messages:

```
Truncation was not explicitely activated but `max_length` is provided a specific value, please use `truncation=True` to explicitely truncate examples to max length. Defaulting to 'longest_first' truncation strategy. If you encode pairs of sequences (GLUE-style) with the tokenizer you can select this strategy more precisely by providing a specific strategy to `truncation`.
```

I think this is related to #482 .

**Comment by tholor** [comment]
Hey, 
For which task do you get this? Are you using fast or slow tokenizers?

**Comment by PhilipMay** [comment]
Text classification. Since I did not specify to use fast I think I use slow.

**Comment by tholor** [comment]
Fixed in #528
